moto/forecast/models.py

```diff
@@ -0,0 +1,183 @@
+from future.utils import iteritems
+import re
+from datetime import datetime
+from enum import Enum
+
+
+from boto3 import Session
+
+from moto.core import ACCOUNT_ID, BaseBackend
+from moto.core.utils import iso_8601_datetime_without_milliseconds
+from .exceptions import (
+    InvalidInputException,
+    ResourceAlreadyExistsException,
+    ResourceNotFoundException,
+    ValidationException,
+)
+
+
+class Domain(Enum):
```

Comment:
The Enum doesn't seem to be in use anymore?

Reply:
Yes. I had it in here originally, but it was proving to be less readable to use the enum. I'll remove it and all references to it.